lib/python/ensembl/brc4/runnable/process_seq_region.py

```diff
@@ -290,17 +305,34 @@ def merge_regions(self,
                 seqr1 = regions1[name]
             if name in names2:
                 seqr2 = regions2[name]
-            
+
+            final_seqr = {}
             if seqr1 and seqr2:
-                merged_seqr = {**seqr1, **seqr2}
-                seq_regions.append(merged_seqr)
+                final_seqr = self.merge_dicts(seqr1, seqr2)
```

Comment:
Here is a nice trick to do the same (shallow) merge (respecting the rule that the second value is preserved over the first one):
```suggestion
                final_seqr = {**seqr1, **seqr2}
```